Scripts/TelligentLocalInstance/TelligentLocalInstance.ps1

```diff
@@ -140,7 +141,8 @@ function Install-TelligentInstance {
         [Parameter(ValueFromPipelineByPropertyName=$true)]
 		[ValidateScript({!$_ -or (Test-Zip $_) })]
         [string] $HotfixPackage,
-        [switch] $WindowsAuth
+        [switch] $WindowsAuth,
+        [string] $DBServerName
```

Comment:
Can we call the parameter `DatabaseServer` rather than `DBServerName` to better match powershell conventions, and consistency with `Install-Community`